pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -107,3 +108,29 @@ def attach_model_logging_functions(self, model):
         for callback in self.trainer.callbacks:
             callback.log = model.log
             callback.log_dict = model.log_dict
+
+    def attach_model_callbacks(self, model: LightningModule) -> None:
```

Comment:
is it meant to be a public interface for the connector?

Reply:
I thought I can leave it public because connectors are internal objects. But making it protected won't hurt :)